src/test/java/echoServer/EchoerTest.java

```diff
@@ -34,10 +34,10 @@ void testReadClientInputCallsPrintWriterPrintLn() throws IOException {
 
         String expectedResult = StatusCode.PAGE_NOT_FOUND.httpResponse;
 
-        String input = "GET / HTTP/1.1\r\n" +
-                "Host: localhost:5000\r\n" +
+        String input = "Accept: */*\r\n" +
                 "User-Agent: curl/7.64.1\r\n" +
-                "Accept: */*\r\n\r\n";
+                "Host: localhost:5000\r\n" +
+                "GET / HTTP/1.1\r\n\r\n";
```

Comment:
A bit of a nitpick here, but technically when you're running these tests, your user agent is not `curl`